docs/ibc/overview.md

```diff
@@ -1,180 +1,150 @@
-<!--
-order: 1
--->
+<!-- order: 1 -->
 
-# Overview
+ # IBC Overview
 
-Learn what IBC is, its components and use cases. {synopsis}
+Learn what IBC is, its components, and use cases. {synopsis}
 
-## What is the Interblockchain Communication Protocol (IBC)?
+## What is the Inter-Blockchain Communication Protocol (IBC)?
```

Comment:
let's use the Inter-Blockchain Communication protocol (IBC) 
capitalize I and B and C and use a hyphen